Travel-list/src/App.jsx

```diff
@@ -1,13 +1,50 @@
+import { useState } from "react";
 import "./App.css";
 import FormSection from "./components/FormSection";
 import Header from "./components/Header";
+import PackingList from "./components/PackingList";
+import Stats from "./components/Stats";
 
 function App() {
+  const [itemObject, setItemObject] = useState([
```

Comment:
`itemObject` should be renamed to `items`.. 

`itemObject` represents a single item..But we are having array of items.. Also `itemsArray` is not a good name, we don't need to tell in the variable name what data structure we are using..